output/elastic/outputelastic.go

```diff
@@ -112,12 +113,19 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeOutputC
 	// map Printf to error level
 	logger := &errorLogger{logger: goglog.Logger}
 
+	// replace env var names with values on URL config
+	for _, url := range conf.URL {
+		newURL := logevent.FormatWithEnv(url)
+		conf.resolvedURLs = append(conf.resolvedURLs, newURL)
```

Comment:
Do we need to reset it before appending urls?

Reply:
conf.resolvedURLs ? 

I don't think so as this comes empty after creating conf in 

```suggestion
conf := DefaultOutputConfig()
err := config.ReflectConfig(raw, &conf)
```
and ReflectConfig does not populate resolvedURLs as it's private.

Reply:
OK